language/libra-vm/src/libra_transaction_executor.rs

```diff
@@ -724,6 +724,7 @@ fn convert_txn_args(args: &[TransactionArgument]) -> Vec<Value> {
             TransactionArgument::Address(a) => Value::address(*a),
             TransactionArgument::Bool(b) => Value::bool(*b),
             TransactionArgument::U8Vector(v) => Value::vector_u8(v.clone()),
+            TransactionArgument::Vector(_) => panic!("not implemented"),
```

Comment:
`unimplemented!()` is for that : o also isn't this adding a potential crash oO? (no idea, haven't looked further)

Reply:
yes, pls don't add panic() in the VM layer :-D

Reply:
this is a POC (see other comments)